give.spec

```diff
@@ -14,7 +14,7 @@
 
 Name: give
 Version: 3.1
-Release: 5%{?dist}
+Release: 5
```

Comment:
Why? Including the Linux OS distribution is pretty standard for rpm packages.

Reply:
The default tarfile name when I `wget` only has the `%{version}`, and release number, not `%{dist}`.